### DISCUSSION TITLE
How to add extensions which are not a part of PHP distribution?

### DISCUSSION THREAD

**Discussion by artello73** [php/frankenphp#462] [Q&A]
Hello,

Considering to give a try of FrankenPHP, but got stuck with the question on how to add custom extension. I have a project which needs Snowflake PDO driver which needs to be built from sources (https://github.com/snowflakedb/pdo_snowflake). Is there any guidelines on how to integrate it to FrankenPHP?

**Comment by withinboredom** [comment]
It uses the same stuff as regular PHP docker images, if you are not using standalone binaries. If the library can be built in ZTS (thread-safe), it will work.
